src/app/app.pri

```diff
@@ -32,8 +32,8 @@ stacktrace {
     else {
         HEADERS += $$PWD/stacktrace_win.h
         !nogui {
-            HEADERS += $$PWD/stacktrace_win_dlg.h
-            FORMS += $$PWD/stacktrace_win_dlg.ui
+            HEADERS += $$PWD/stacktrace_win_dialog.h
```

Comment:
Stacktrace is 3rd party component.

Reply:
Then why does it have the "Bittorrent Client using Qt and libtorrent...". I'll revert it.

Reply:
Oh, sorry, I messed it up. Dialog is ours!
So let it be StacktraceDialog. Without any "win" since it really independent from OS.

Reply:
Roger

Reply:
Wait is it really OS independent?
https://github.com/qbittorrent/qBittorrent/blob/c66738dc0536f514fadb3c5f791a004940831470/src/app/app.pri#L28-L39